.eslintrc.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
   plugins: ["markdown"],
   overrides: [
     {
-      files: ["rollup.config.js", "**/*.md"],
+      files: ["rollup.config.js", "**/*.md/**"],
```

Comment:
ughh ok that makes sense, I assume that lets you target language-specific code blocks in a markdown file

Reply:
nice find!

Reply:
> ughh ok that makes sense, I assume that lets you target language-specific code blocks in a markdown file

yep! i tried it locally, but didn't try extending it to the other configs